ckan/tests/pytest_ckan/fixtures.py

```diff
@@ -319,7 +319,7 @@ def factory(data, filename, context={}, **kwargs):
         action = kwargs.pop(u"action", u"resource_create")
         field = kwargs.pop(u"upload_field_name", u"upload")
         test_file = BytesIO()
-        test_file.write(six.ensure_binary(data))
+        test_file.write(bytes(data))
```

Comment:
@steveoni Try to add this here to fix the current failures

```suggestion
        test_file.write(bytes(data, encoding='utf8'))
```